.gitignore

```diff
@@ -19,3 +19,8 @@ dist/*
 # you can add these to your own "global" .gitignore file.
 # https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
 #
+# Environments
+.env
+.venv
+env/
+venv/
```

Comment:
These should go in your own "global" .gitignore file; it's not necessary to include them here (note the comment starting on line 17).